litestar/_signature/models/pydantic_signature_model.py

```diff
@@ -56,7 +56,7 @@ def to_dict(self) -> dict[str, Any]:
 
         Returns: A dictionary of string keyed values.
         """
-        return {key: self.__getattribute__(key) for key in self.__fields__}
+        return {key: self.__getattribute__(key) for key in self.model_fields}
 
     @classmethod
     def signature_field_from_model_field(cls, model_field: ModelField) -> SignatureField:
```

Comment:
This change to `ModelField` breaks all subsequent attribute references below... But i cant find equivalents